### PR TITLE
LPD-23028 | master

### PR DESCRIPTION
**Pull request by fmarcio**
https://liferay.atlassian.net/browse/LPD-23028

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -1564,7 +1564,7 @@ well-send-you-an-email-once-its-ready-to-access-at-this-url-x=We will send you a
 weve-found-multiple-accounts-associated-with-x-.-you-can-have-one-basic-tier-workspace-of-analytics-cloud-per-account.-please-associate-this-analytics-cloud-workspace-to-an-account=We have found multiple accounts associated with {0}. You can have one basic tier Workspace of Analytics Cloud per account. Please associate this Analytics Cloud Workspace to an account.
 when-either-limit-is-exceeded-the-current-plan-will-either-have-to-be-upgraded-or-add-ons-will-have-to-be-purchased-to-accommodate-the-overage=When either limit is exceeded, the current plan will either have to be upgraded or add-ons will have to be purchased to accommodate the overage.
 when-either-limit-is-exceeded-the-current-plan-will-have-to-be-upgraded-to-business-or-enterprise=When either limit is exceeded, the current plan will have to be upgraded to Business or Enterprise.
-where-fragment=where
+where-fragment=where attribute
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/resources/content/Language.properties:1567]: `where-attribute-fragment`

**Review by interaminense — CHANGES_REQUESTED** [review]

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:stable

**Comment by interaminense** [comment]
ci:forward:force

**Pull request closed without merging**